godot/src/decentraland_components/avatar/avatar.gd

```diff
@@ -3,18 +3,18 @@ extends DclAvatar
 
 signal avatar_loaded
 
+# Debug to store each avatar loaded in user://avatars
```

Comment:
Default should be FALSE, and this can be added as another script, and connecting with a signal.

Reply:
changed to `false` and created https://github.com/decentraland/godot-explorer/issues/378